microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/JoseHeader.java

```diff
@@ -0,0 +1,91 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+
+/**
+ * A class representing a header of JWT
+ */
+public class JoseHeader {
```

Comment:
Why do we need this file?

Isn't available any standard class for this ?

How will customers do this stuff ? If each of them will do the same WF/EAP should provide it for them.

Reply:
Users will use ready made tooling created with specification in mind. We need to do negative test cases too and that's something which can't be done with tooling which validates against spec.